### DISCUSSION TITLE
Works fine with Norwii N95

### DISCUSSION THREAD

**Discussion by ossk** [jahnf/Projecteur#180] [General]
Dear Jahn,

thanks a lot for the effort to create projecteur. I just wanted to let you know that it works perfectly with the cheap Norwii N95 Spotlight, vendorId 3243, deviceId 0122 after adding the udev rule and by explicitly calling with the device ID (-D).

Cheers
    Volker

**Comment by jahnf** [comment]
Thank you Volker for letting me know, I didn't know about this device.
